assets/html/archive.html

```diff
@@ -79,7 +79,7 @@
           </li>
         </ul>
       </div>
-      <% if (isOwner) { %>
+      <% if (isOwner || sessionUser.scopes.includes('admin:dats')) { %>
```

Comment:
This doesn't work, unfortunately. The /v1/archives/remove route operates on the session user.

We should use a separate link entirely, that indicates it's an admin action, and we should add a route to the admin API in hypercloud which removes the archive.